packages/components/src/alignment-matrix-control/test/index.tsx

```diff
@@ -37,7 +39,9 @@ describe( 'AlignmentMatrixControl', () => {
 					<AlignmentMatrixControl value="center" onChange={ spy } />
 				);
 
-				await act( () => getCell( alignment ).focus() );
+				await user.click( getCell( alignment ) );
+
+				expect( getCell( alignment ) ).toHaveFocus();
```

Comment:
Replacing a `.focus()` with a `.click()` makes the intention a bit less clear, but the "have focus" assertion definitely helps clarify it, so thanks for adding that assertion 👍

Reply:
Thank you for sharing! It's helpful to know it was a useful addition.